src/ui/units/wizard/components/Dialogs/DialogColor/DialogColor.tsx

```diff
@@ -191,10 +194,11 @@ class DialogColorComponent extends React.Component<Props, State> {
                 polygonBorders,
                 reversed,
                 thresholdsMode,
-                leftThreshold,
-                middleThreshold,
-                rightThreshold,
+                leftThreshold: leftThreshold ? leftThreshold : undefined,
+                middleThreshold: middleThreshold ? middleThreshold : undefined,
+                rightThreshold: rightThreshold ? rightThreshold : undefined,
```

Comment:
As far as I understand it, it works now because the values are set in strings, right? But it is better to explicitly check for undefined if this is the goal - in this case, 0 will not be lost when the values become numbers (as it should be)